aten/src/TH/THStorageClass.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
```

Comment:
Want to add a comment to this file saying why we've split the Class and Functions?